internal/cuetools/testdata/assertionmodes/tests/incorrect_unification_extra.cue

```diff
@@ -0,0 +1,15 @@
+
+@if(incorrect_unification_extra)
+package tests
+
+#request: observed: composite: resource: {
+	foo: "bar"
+}
+
+#mode: "unification"
```

Comment:
This was left in as an omission, based on thinking around whether it would be nicer to represent it as a definition instead of an attribute. An attribute leads itself nicer to partial unification as mentioned in another top-level PR comment.